ui/src/components/member/AddMember.js

```diff
@@ -182,7 +183,23 @@ class AddMember extends React.Component {
     }
 
     inputChanged(key, evt) {
-        this.setState({ [key]: evt.target.value });
+        this.setState({ [key]: evt ? evt.value : '' });
+    }
+
+    userSearch(part) {
+        if (part.startsWith('user.')) {
```

Comment:
- I believe `'user.'` needs to be configurable coming from constants.USER_DOMAIN
- we might want to keep an option to extend this feature for other types of principals ( headless users initially may be services down the road if its possible )

Reply:
basically, it is just backward compatibility. 
currently there is no need to type the prefix (user.) , since it is autocomplete by the user short id.
extend this piece of code might be very easy and I don't see a reason to make it more complex now ...